Sources/RevolutionKit/Rewriter/XCTestRewriter+MethodVisitor.swift

```diff
@@ -111,10 +111,10 @@ extension XCTestRewriter {
         }
     }
     
-    /// Returns true if the method is static
-    private func isStaticMethod(node: FunctionDeclSyntax) -> Bool {
+    /// Returns true if the method is static or private
+    private func isStaticOrPrivateMethod(node: FunctionDeclSyntax) -> Bool {
         node.modifiers.contains {
-            $0.tokens(viewMode: .sourceAccurate).contains { $0.tokenKind == .keyword(.static) }
+            $0.name.text == "static" || $0.name.text == "private"
```

Comment:
It's better to use `.keyword(.static)` and `.keyword(.private)` for strictency. Could you keep the original implementation as possible?

Reply:
Sure. I went with `$0.name.tokenKind == .keyword(...)`. The `.tokens(viewMode: .sourceAccurate)` is not needed, however, as there's only a single token anyway.